docs/design/INTEGRATE-ADMINAPI.V1.md

```diff
@@ -0,0 +1,488 @@
+* `InstanceContext.cs` - Instance context management
+* `OdsSecurityVersionResolver.cs` - Security version resolution (remove from V1)
+* `OperationalContext.cs` - Operational context management
+* `ValidatorExtensions.cs` - Validation helper extensions
+* `WebApplicationBuilderExtensions.cs` - Application builder extensions
+* `WebApplicationExtensions.cs` - Application configuration extensions
+* **Security folder and all classes** - Use V2 security implementation
+* **Connect\Register and Connect\Token endpoints** - Use V2 implementation
+* **Artifacts folder** - Remove from V1, use V2 artifacts
+* **Information feature** - Remove from V1, use V2 implementation
+
+### 2.2 DataAccess Layer Strategy
+
+**Objective**: Maintain V1 compatibility by preserving Ed-Fi ODS 6.x DataAccess implementations.
+
+**Tasks**:
+
+* Copy Ed-Fi ODS 6.x Admin.DataAccess and Security.DataAccess code implementation files directly to AdminAPI V1 project
+* Avoid handling divergence between 6.x and 7.x Security and Admin DataAccess usages while maintaining V1 API compatibility
+  
+* **Isolation Benefits**:
+  * V1 maintains stable DataAccess layer independent of V2 upgrades
+  * Eliminates version compatibility complexity in shared DataAccess components
+  * Reduces risk of breaking V1 functionality when V2 adopts newer Ed-Fi ODS versions
+  
+### 2.4 Database Setup Strategy
+
+**Objective**: Maintain separate database infrastructure to support V1 with
+Ed-Fi ODS 6.x and V2 with Ed-Fi ODS 7.x DataAccess compatibility.
+
+**Tasks**:
+
+* **Separate Database Instances**: Setup dedicated EdFi_Admin and EdFi_Security databases for each version:
+  * `EdFi_Admin_V1` and `EdFi_Security_V1` (6.x schema)
+  * `EdFi_Admin` and `EdFi_Security` (7.x schema)
+
+* **Version-Specific Connection Strings**: Define separate connection string configurations for V1 and V2 since each uses different DbContexts:
+
+  ```json
+  {
+        "ConnectionStrings": 
+        {
+            "v1": {
+                // V1 connections (6.x schema)
+                "EdFi_Admin": "Server=.;Database=EdFi_Admin_V1;Integrated Security=true",
+                "EdFi_Security": "Server=.;Database=EdFi_Security_V1;Integrated Security=true",
+            },
+            "v2": {
+                // V2 connections (7.x schema) 
+                "EdFi_Admin": "Server=.;Database=EdFi_Admin;Integrated Security=true",
+                "EdFi_Security": "Server=.;Database=EdFi_Security;Integrated Security=true"
+            }
+        }
+   }
+  ```
+
+* **DbContext Registration**: Configure separate DbContext instances in DI container:
+
+  ```csharp
+  // V1 DbContext (6.x DataAccess)
+  services.AddDbContext<AdminApiV1DbContext>(options =>
+      options.UseSqlServer(connectionString.GetConnectionString("EdFi_Admin_V1")));
+  
+  // V2 DbContext (7.x DataAccess) 
+  services.AddDbContext<AdminApiDbContext>(options =>
+      options.UseSqlServer(connectionString.GetConnectionString("EdFi_Admin")));
```

Comment:
With the new "mode" appsetting, now we can just choose which DbContext to setup.